easybuild/easyconfigs/d/darshan-util/darshan-util-3.4.4-GCC-12.3.0.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'ConfigureMake'
+
+name = 'darshan-util'
+version = '3.4.4'
+
+homepage = 'https://www.mcs.anl.gov/research/projects/darshan/'
+description = """Darshan, a scalable HPC I/O characterization tool. Darshan is 
+designed to capture an accurate picture of application I/O behavior, including 
+properties such as patterns of access within files, with minimum overhead.  
+The name is taken from a Sanskrit word for 'sight' or 'vision'. darshan-util
+can analyze log files produced by darshan-runtime"""
+
+toolchain = {'name': 'GCC', 'version': '12.3.0'}
+
+source_urls = ['https://web.cels.anl.gov/projects/darshan/releases']
+sources = ['darshan-%(version)s.tar.gz']
+checksums = ['d9c9df5aca94dc5ca3d56fd763bec2f74771d35126d61cb897373d2166ccd867']
+
+start_dir = 'darshan-util'
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('Autotools', '20220317'),
+]
+
+dependencies = [
+    ('Perl-bundle-CPAN', '5.36.1'),
+    ('texlive', '20230313'),
+    ('gnuplot', '5.4.8'),
```

Comment:
texlive is a painful dep with a gazillion files. If it all possible, if the software can function to any degree without latex, i would say to just have this opt-in at runtime. If not, then disregard this comment and keep texlive in.